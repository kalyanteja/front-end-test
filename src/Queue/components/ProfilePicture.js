import styled from 'styled-components';

export default styled.div`
  background-color: grey;
  border-top-left-radius: 0.2em;
  border-top-right-radius: 0.2em;

  @media (max-width: 450px){
    height: 290px;
    width: 290px;
    background-image: ${props => `url(https://www.gravatar.com/avatar/${props.imageHash}?s=300)`};
  }
  
  @media (min-width: 450px){
    height: 170px;
    width: 170px;
    background-image: ${props => `url(https://www.gravatar.com/avatar/${props.imageHash}?s=200)`};
  }
`;
