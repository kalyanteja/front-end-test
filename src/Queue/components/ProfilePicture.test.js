import React from 'react';
import ProfilePicture from './ProfilePicture';
import renderer from 'react-test-renderer';

describe('picture component testing', () => {
  it('renders picture styled-component', () => {
    const tree = renderer
    .create(<ProfilePicture></ProfilePicture>)
    .toJSON();
    expect(tree).toMatchSnapshot()
  });
})

