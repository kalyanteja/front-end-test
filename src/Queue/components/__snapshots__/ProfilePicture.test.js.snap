// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`picture component testing renders picture styled-component 1`] = `
<div
  className="sc-bdVaJa gAxlWe"
/>
`;
